### PR TITLE
LocalDate and LocalTime Parse() implementation

### PR DESCRIPTION
**Pull request by jorgebay**

### CODE REVIEW

File: src/Cassandra/LocalTime.cs [hunk 1]
```diff
@@ -100,6 +97,32 @@ public LocalTime(int hour, int minute, int second, int nanosecond)
                 nanosecond;
         }
 
+        /// <summary>
+        /// Creates a new <see cref="LocalDate"/> instance based on the string representation.
```
Comment (landim) [inline src/Cassandra/LocalTime.cs]: typo: should be LocalTime

File: src/Cassandra/LocalTime.cs [hunk 2]
```diff
@@ -100,6 +97,32 @@ public LocalTime(int hour, int minute, int second, int nanosecond)
...
+        public static LocalTime Parse(string value)
+        {
+            if (value == null)
+            {
+                throw new ArgumentNullException(nameof(value));
+            }
+            var parts = value.Split(':');
+            if (parts.Length < 2 || parts.Length > 3)
+            {
+                throw new FormatException("LocalDate format is invalid");
```
Comment (landim) [inline src/Cassandra/LocalTime.cs]: same: LocalDate instead of LocalTime

File: src/Cassandra/LocalTime.cs (hunk 1 above)
Comment (jorgebay) [inline src/Cassandra/LocalTime.cs]: Nice catch!

**Comment by jorgebay** [comment]
Fixed xmldoc and exception typos, I'll merge it as soon as unit tests on CI builds finish.

**Pull request merged**